packages/contracts/hardhat.config.ts

```diff
@@ -39,6 +39,7 @@ extendEnvironment((hre: HardhatRuntimeEnvironment) => {
   const testingFork: TestingFork = {
     network: '',
     osxVersion: '',
+    activeContracts: {},
```

Comment:
Is it always correct to initialize the testing fork with empty active contracts?

Reply:
yes,
bcoz if u want to populate it u can at any time during ur tests